data/i18n/ar/ar.toml

```
@@ -0,0 +1,561 @@
+# i18n strings for the English (main) site.
+# NOTE: Please keep the entries in alphabetical order when editing
+
+# For "and", see [conjunction_1]
+
+[auto_generated_edit_notice]
+other = "(صفحة مولدة آلياً)"
+
+[auto_generated_pageinfo]
+other = """<p>هذه الصفحة مولدة آلياً،</p>
+<p>إذا أردت الإبلاغ عن مشكلة في هذه الصفحة، أشر إلى أنها تم توليدها آلياً عند وصف مشكلتك، لاحتمال وجود سبب المشكلة في ملف آخر في مشروع كوبرنيتيز.</p>"""
+
+[caution]
+other = "تحذير:"
+
+[cleanup_heading]
+other = "جاري الترتيب"
+
+[community_events_calendar]
+other = "تقويم الفعاليات"
+
+[community_forum_name]
+other = "المنتدى"
+
+[community_github_name]
+other = "(GitHub) جيت هاب"
+
+[community_slack_name]
+other = "(Slack) سلاك"
+
+[community_stack_overflow_name]
+other = "(Stack Overflow) ستاك أوفرفلو"
+
+[community_twitter_name]
+other = "(Twitter) تويتر"
+
+[community_youtube_name]
+other = "(YouTube) يوتيوب"
+
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
+[conjunction_1]
+other = "و"
+
+[cve_id]
+other = "CVE ID"
+
+[cve_issue_url]
+other = "رابط القضية للCVE"
+
+[cve_summary]
+other = "ملخص القضية"
+
+[cve_table]
+other = "القائمة الرسمية للCVE الخاصة بكوبرنيتيز"
+
+[cve_table_date_format]
+other = "02 Jan 2006 15:04:05 MST"
+
+[cve_table_date_format_string]
+other = "(آخر تحديث: %s)"
+
+[deprecation_title]
+other = "تعرض الآن وثائق كوبرنيتيز للنسخة الآتية:"
+
+[deprecation_warning]
+other = " خارج عن الخدمة، ولذلك قد تكون المعلومات في هذه الوثيقة قديمة. للاطلاع على أحدث المعلومات الرجاء رؤية "
```

```suggestion
other = " خارج عن الخدمة، لذلك قد تكون المعلومات في هذه الوثيقة قديمة. للاطلاع على أحدث المعلومات الرجاء رؤية "
```
أعتقد أن تجنب حروف العطف (الواو) و(الفاء) و(ثم) .. يكون النص أكثر رسمية

---

شكراً، تم في https://github.com/kubernetes/website/pull/44862/commits/7b2b2d3af38528854b53bd971862176ce4717e40.